src/app/error.tsx

```diff
@@ -0,0 +1,20 @@
+"use client";
+import Error from "@/components/Error";
+
+type ErrorProps = {
+  code: number;
+  error: string;
+  name: string;
+  message: string;
+  dev: string;
+};
+
+type Props = {
```

Comment:
integrate the error with with ErrorProps, u should just need a single type for this, u can also call the ErrorProps just props